DigitalLearningSolutions.Data.Tests/DataServices/CourseAdminFieldsDataServiceTests.cs

```diff
@@ -169,5 +169,21 @@ public void GetDelegateAnswersForCourseAdminFields_returns_expected_results()
                 result.Should().BeEquivalentTo(expectedResult);
             }
         }
+
+        [Test]
+        public void GetCourseFieldPromptIdsForCustomisation_returns_expected_results()
+        {
+            // Given
+            var expectedResult = new int[] { 1, 2, 0 };

```

Comment:
The `int` type is redundant here, since its inferred from the values

Reply:
So: `new [] { 1, 2, 0 }` ?

Reply:
Yep 🙂 https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/arrays/implicitly-typed-arrays